src/inline-svg.directive.ts

```diff
@@ -141,7 +141,7 @@ export class InlineSVGDirective implements OnInit, OnChanges, OnDestroy {
   private _insertEl(el: Element) {
     if (this.replaceContents && !this.prepend) {
       if (this._prevSVG) {
-        this._prevSVG.remove();
+        this._el.nativeElement.removeChild(this._prevSVG);
```

Comment:
I guess just for the sake of not relying on the fact that he SVG is in that element (which in theory it should be), we could just do `this._prevSVG.parentElement...`.